### PR TITLE
build: move proto test files to common place

### PR DESCRIPTION
**Pull request by octonato**
This is in preparation for full generation tests for the Scala SDK. 

We have a bunch of files used by `codegenJavaCompilationTest` that exercise many different options. 

As we progress with the codegen for Scala, we want the Scala codegen to run on those same files.

### CODE REVIEW

**Comment by raboof** [comment]
> This is in preparation for full generation tests for the Scala SDK.

Don't we already have full generation tests, the scripted tests of the sbt plugin?

**Comment by octonato** [comment]
> Don't we already have full generation tests, the scripted tests of the sbt plugin?

My sentence is not clear. Yes, we do have full generation. What I mean, we should be able to have exhaustive generations to be able to catch corner cases. Like usage of outer java class, multi-files, import from other packages, etc. That's why we have this list of files.

File: build.sbt [hunk 1]
```diff
@@ -171,6 +171,7 @@ lazy val codegenJavaCompilationTest = project
   .settings(
     (publish / skip) := true,
     name := "akkaserverless-codegen-java-compilation-tests",
+    Compile / PB.protoSources += baseDirectory.value / ".." / ".." / "proto-test-files",
```
Comment (patriknw) [inline build.sbt:174]: Are we going to do something similar like this in the scripted test to include them? Have you tried that it works?

**Review by patriknw — COMMENTED** [review]
LGTM, with a question about how to use them...

**Review by patriknw — APPROVED** [review]
